src/views/Header.svelte

```diff
@@ -4,7 +4,8 @@
 			<Logo size="5em" />
 			<span>Metis</span>
 		</div>
-		<div slot="right" class="menu_holder" on:click={ToggleMenu}>
+		<!-- svelte-ignore a11y-click-events-have-key-events -->
+		<div slot="right" class="menu" on:click={ToggleMenu}>
```

Comment:
@GlistenSTAR this has created troubles... I presented `menu_holder` for a reason.